app/views/audits/_table.html.erb

```diff
@@ -52,6 +52,6 @@
   </tbody>
 </table>
 
-<%= paginate audits if audits.respond_to?(:total_pages) %>
+<%= paginate @pagy, audits if @pagy %>
```

Comment:
Why do we pass the collection into `paginate`? It looks like it only uses the `pagy` object? Could maybe change this method to simply `paginate` which would only take `@pagy`, or expect an instance var `@pagy` to be defined and raise if not?

Reply:
copy-pasted from cerebro where we say "15 Users" ... but I guess that's kinda useless anyway ... removing